pom.xml

```diff
@@ -68,27 +52,30 @@
     	    <version>2.7.0</version>
     	    <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.commons.osgi</artifactId>
-            <version>2.1.0</version>
-            <scope>provided</scope>
-        </dependency>
         <dependency>
             <groupId>org.osgi</groupId>
-            <artifactId>org.osgi.compendium</artifactId>
-        </dependency>
-        <dependency>
-        	<groupId>org.apache.felix</groupId>
-        	<artifactId>org.apache.felix.scr.annotations</artifactId>
+            <artifactId>osgi.cmpn</artifactId>
```

Comment:
Please switch to individual OSGi dependencies (https://docs.osgi.org/artifacts/)

Reply:
Fixed 👍